_wasm_crypto/crypto.js

```diff
@@ -0,0 +1,344 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+// deno-lint-ignore-file
```

Comment:
why? If it is because this is essentially a code-gen file (which I believe it is) then we should put a note here to that effect.